src/main/java/mate.academy/service/ProductService.java

```diff
@@ -1,8 +1,10 @@
 package mate.academy.service;
 
 import java.util.List;
+import mate.academy.lib.Component;
 import mate.academy.model.Product;
 
+@Component
 public interface ProductService {
```

Comment:
should this annotation be placed over interface or implementation?

Reply:
I don`t know why , but when I delete this annotation , I get an exception(